css/properties/margin-bottom.json

```diff
@@ -0,0 +1,104 @@
+{
+  "css": {
+    "properties": {
+      "margin-bottom": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/margin-bottom",
+          "support": {
+            "webview_android": {
+              "version_added": "1"
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": "1"
```

Comment:
There hasn't been a Fx for android version 1: put 4 (version 1 existed only for Maemo).

Florian has a PR that enforces valid versions number with the linter. This will be catched by it in the future.